### PR TITLE
ms: fix label on storageclassrequest cr

### PR DESCRIPTION
**Pull request by subhamkrai**
Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2188666

### CODE REVIEW

**Comment by subhamkrai** [comment]
@Nikhil-Ladha @iamniting

File: services/provider/server/storageclassrequest.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (s *storageClassRequestManager) Create(
 			Namespace: s.namespace,
 			Labels: map[string]string{
 				controllers.ConsumerUUIDLabel: consumerUUID,
-				storageClassRequestName:       storageClassRequestName,
+				storageClassRequestName:       storageClassRequestNameLabel,
```
Comment (iamniting) [inline services/provider/server/storageclassrequest.go:73]: I think it should be vice versa @Madhu-1 WDYT?

**Review by iamniting — CHANGES_REQUESTED** [review]

File: services/provider/server/storageclassrequest.go (hunk 1 above)
Comment (Madhu-1) [reply services/provider/server/storageclassrequest.go:73]: Yes it should always be key:value  key is the const identifier and value should be the request name

Comment (iamniting) [reply services/provider/server/storageclassrequest.go:73]: @subhamkrai Can you pls change it as Madhu as confirmed it?

Comment (subhamkrai) [reply services/provider/server/storageclassrequest.go]: right 😅

**Review by Madhu-1 — APPROVED** [review]

**Review by iamniting — APPROVED** [review]

**Comment by Madhu-1** [comment]
we shouldnot use `bug 2188666` in the PR title as the fix is getting into main branch it's only required for backporting or else Bugzilla will update the bug once this PR gets merged in to the main branch.

**Comment by subhamkrai** [comment]
> we shouldnot use `bug 2188666` in the PR title as the fix is getting into main branch it's only required for backporting or else Bugzilla will update the bug once this PR gets merged in to the main branch.

okay, got it.

**Pull request merged**